src/utils/export/CsvData.ts

```diff
@@ -21,7 +21,7 @@ const csvFields = [
   },
   {
     label: 'Created By',
-    value: 'createdBy'
+    value: 'createdByUserAccount.commonName'
```

Comment:
Should we also make this change in the queries & this file for `modifiedDts`?
Looks like we do have a [modifiedByUserAccount](https://github.com/CMSgov/mint-app/blob/78e5debd8e78a778ba4b03480d42d84d4ca79551/pkg/graph/schema.graphql#L58) in the schema, but what I'm not sure about is what happens if we use that and `modifiedByUserAccount` is `null` (which it can be if nothing's been modified - `scripts/dev db:seed` creates a few like this!)

I'm thinking we should, since this is what renders in the column as-is
![image](https://github.com/CMSgov/mint-app/assets/15203744/0e5abd3d-036f-4a55-b843-bb9e44318d96)

Reply:
Honestly thinking of nixing this branch/PR as the subsequent PR is kinda a rework of the csv fields and will be replaced anyway/merge conflicts